lib/ello_click/affiliate.ex

```diff
@@ -12,7 +12,7 @@ defmodule ElloClick.Affiliate do
   defp affiliate(conn) do
     conn.assigns.url
     |> skip_ello
```

Comment:
TBH, I was thinking you could just add a `skip_vimeo` function here, it just needs to check the url and return `{:ok, url}` if it was vimeo, and just `url` if not.

Reply:
@alanpeabody Done 👌